packages/framework/src/Controller/Admin/FeedController.php

```diff
@@ -135,7 +135,7 @@ public function listAction(): Response
                     'feedName' => $feedInfo->getName(),
                     'domainConfig' => $domainConfig,
                     'url' => $this->feedFacade->getFeedUrl($feedInfo, $domainConfig),
-                    'created' => $feedTimestamp === null ? null : (new DateTime())->setTimestamp($feedTimestamp),
+                    'created' => $feedTimestamp === null ? null : (new DateTimeImmutable())->setTimestamp($feedTimestamp),
                     'generate' => null,
```

Comment:
What about `new DatePoint` everywhere instead of `new DateTimeImmutable`?
That way we can improve testability, because we would be able to set global clock in test